NuGet.config

```diff
@@ -4,6 +4,7 @@
     <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
     <add key="applicationinsights" value="https://www.myget.org/F/applicationinsights/api/v3/index.json" />
+    <add key="Local" value="D:\dev\github\ai\bin\Debug\Nuget" />
```

Comment:
This change will **not** be submitted. Only here for local development work while I wait for -dotnet-server to get posted to myget.

Reply:
I've removed this now, and have updated the dependencies to a myget package representing a more valid candidate for the next beta release here.